### DISCUSSION TITLE
Will Create3 support ROS Jazzy?

### DISCUSSION THREAD

**Discussion by LucasCosta-StreetDrone** [iRobotEducation/create3_docs#580] [Questions] [answered]
### How are you connecting to your Create 3?

Wi-Fi (ROS 2)

### Computer(s) Model(s) and Operating System(s)

Raspberry Pi 4

### Which version of ROS 2 is installed on your computer?

Humble

### Which firmware version is installed on your robot?

H.2.6

### Which RMW is your robot running?

CycloneDDS

### Does your robot have an assigned namespace? If so, please share.

/bob

### Is the robot connected to a network? If so, what is the network type?

Yes; home wifi

### Are there multiple Create 3 robots connected to your network?

No

### Is multicast enabled?

Yes

### What is the Adapter Board's USB/BLE Toggle currently switched to?

USB

### Describe your question.

I suppose a firmware update will be needed for those who desire to work with [ROS Jazzy](https://docs.ros.org/en/iron/Releases/Release-Jazzy-Jalisco.html).

Do you plan to support it?

If yes, is there an estimate of when it will be released?

Thanks in advance.

**Comment by mauriz** [comment]
Any update from iRobot about if/when supporting Jazzy? Thanks

**Comment by github-kyle** [comment]
I am also interested in whether Create3 will support Jazzy

**Comment by jhubberts** [comment]
Any word on this?

**Comment by shamlian** [comment, marked as answer]
iRobot [recently released an Iron release](https://github.com/iRobotEducation/create3_docs/releases/tag/I.0.0) for the Create 3 robot which **should be compatible with nodes running ROS 2 Jazzy**.
